utils/db-generator/query_db_generator.cpp

```diff
@@ -0,0 +1,1176 @@
+#include <boost/algorithm/string.hpp>
+#include <cstdlib>
+#include <iostream>
+
+#include <Parsers/ASTFunction.h>
+#include <Parsers/ASTIdentifier.cpp>
+#include <Parsers/ASTIdentifier.h>
+#include <Parsers/ASTLiteral.h>
+#include <Parsers/ASTSelectQuery.h>
+#include <Parsers/ASTTablesInSelectQuery.h>
+#include <Parsers/ASTWithAlias.h>
+#include <Parsers/ParserQueryWithOutput.h>
+#include <Parsers/formatAST.h>
+#include <Parsers/parseQuery.h>
+
+using column_type = uint32_t;
+using table_a_column = std::pair<std::string, std::string>;
+std::string random_string(size_t length)
+{
+    auto randchar = []() -> char
+    {
+        const char charset[] = "0123456789" "ABCDEFGHIJKLMNOPQRSTUVWXYZ" "abcdefghijklmnopqrstuvwxyz";
+        const size_t max_index = (sizeof(charset) - 1);
+        return charset[rand() % max_index];
+    };
+    std::string str(length, 0);
+    std::generate_n(str.begin(), length, randchar);
+    return str;
+}
+std::string random_integer(unsigned int max = 4294967295)
+{
+    int r = rand() % max;
+    return std::to_string(r);
+}
+std::string random_float(int max = 4294967295)
```

Comment:
Maybe `unsigned int` as above? I just have troubles with clang-9.

Reply:
you are right